cmd/validatepcap/validate.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+func isTranscriptValid(transcript Transcript, testCase string) bool {
```

Comment:
I would suggest changing this signature to `func validateTranscript(transcript Transcript, testCase string) error`  and returning `nil` if the transcript is valid and an error otherwise. Right now, It's not clear form the output what went wrong.

Reply:
Done.